src/System/Console/GetOpt/Generics/Modifier.hs

```diff
@@ -3,12 +3,16 @@
 
 module System.Console.GetOpt.Generics.Modifier (
   Modifier(..),
-  deriveShortOptions,
+  Modifiers,
+  mkModifiers,
   mkShortOptions,
-  mkLongOptions,
+  mkLongOption,
+
+  deriveShortOptions,
  ) where
 
 import           Data.List
+import qualified Data.Map.Strict                                as M
```

Comment:
@sol: Would you remove this dependency? (Should be very easy, but it also comes with ghc.)